src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
```

Comment:
According to the checklist, you should not use raw types. Please ensure that the `MachineProducer` interface is parameterized with a bounded type to prevent the creation of implementations that do not use `Machine` or its subtype.